layers/meta-tegrademo/recipes-bsp/u-boot/u-boot-tegra/0015-t210-mender-FIT-image-bootcmd-support.patch

```diff
@@ -0,0 +1,107 @@
+From d1428fef6bd84bc1c95351cb179178c60011869b Mon Sep 17 00:00:00 2001
+From: Ilies CHERGUI <ilies.chergui@gmail.com>
+Date: Sun, 17 Jan 2021 19:34:18 +0000
+Subject: [PATCH] tegra210: mender FIT image bootcmd support.
+
+Signed-off-by: Ilies CHERGUI <ilies.chergui@gmail.com>
+---
+ include/configs/tegra210-common.h |  7 +++++--
+ include/env_default.h             |  7 +++++--
+ include/env_mender.h              | 18 ++++++++++++++++++
+ scripts/config_whitelist.txt      |  1 +
+ 4 files changed, 29 insertions(+), 4 deletions(-)
+
+diff --git a/include/configs/tegra210-common.h b/include/configs/tegra210-common.h
+index 35cead04de..ab38a7e44f 100644
+--- a/include/configs/tegra210-common.h
++++ b/include/configs/tegra210-common.h
+@@ -41,6 +41,8 @@
+  *   for the FDT/DTB to be up to 1M, which is hopefully plenty.
+  */
+ #define CONFIG_LOADADDR 0x80080000
++/* FIT Image Load address */
++#define CONFIG_FITFDT_LOADADDR 0x90000000
+ #define MEM_LAYOUT_ENV_SETTINGS \
+ 	"scriptaddr=0x90000000\0" \
+ 	"pxefile_addr_r=0x90100000\0" \
+@@ -81,8 +83,9 @@
+ 		"/chosen/uuid:" \
+ 		"/chosen/linux,initrd-start:" \
+ 		"/chosen/linux,initrd-end:" \
+-		"/psci/nvidia,system-lp0-disable:" \
```

Comment:
Did you delete this line on purpose?

Reply:
Yes, I did 
I will double check I guess I got an issue if I'm not removing this one.

Reply:
Here is the issue I got when using the FIT and no updates in `fdt_copy_prop_paths`

```
## Loading fdt from FIT Image at 90000000 ...
   Using 'conf@_ddot___ddot___ddot___ddot__nvidia_platform_t210_porg_kernel-dts_tegra210-p3448-0000-p3449-0000-a02.dtb' configuration
   Trying 'fdt@_ddot___ddot___ddot___ddot__nvidia_platform_t210_porg_kernel-dts_tegra210-p3448-0000-p3449-0000-a02.dtb' fdt subimage
     Description:  Flattened Device Tree blob
     Type:         Flat Device Tree
     Compression:  uncompressed
     Data Start:   0x90d8d1a8
     Data Size:    241338 Bytes = 235.7 KiB
     Architecture: AArch64
     Load Address: 0x80000000
     Hash algo:    sha256
     Hash value:   edaa912e1574058c749dc61a1cd9cbd60928f1d183de2d9e7828784f082c86ed
   Verifying Hash Integrity ... sha256+ OK
   Loading fdt from 0x90d8d1a8 to 0x80000000
   Booting using the fdt blob at 0x80000000
   Uncompressing Kernel Image
ERROR: reserving fdt memory region failed (addr=0 size=0)
ERROR: reserving fdt memory region failed (addr=0 size=0)
   Using Device Tree in place at 0000000080000000, end 000000008003deb9
copying carveout for /host1x@50000000/dc@54200000...
copying carveout for /host1x@50000000/dc@54240000...
DT property /psci/nvidia,system-lp0-disable missing in source; can't copy

Starting kernel ...

```

What do you think ? 

**Note:** the boot of the target was successfully